index.template.html

```diff
@@ -2295,8 +2295,8 @@ <h3><code>securityDefinitions</code> and <code>security</code></h3>
   for example, perhaps only those in an administrative role are authorized to perform "special" interactions.
   Tokens can have more than one scope.  
   In this example, an administrator would
-  probably be issued tokens with both the <code>limited</code> and <code>special</code> scopes,
-  while ordinary users would only be issued tokens with the <code>limited</code> scope.
+  probably issue tokens with both the <code>limited</code> and <code>special</code> scopes,
```

Comment:
same problem as above.  "be issued" was correct, IMO.  Or, use the alternative wording mentioned above.